deepchem/models/tensorgraph/graph_layers.py

```diff
@@ -799,3 +841,258 @@ def DAGgraph_step(self, batch_inputs, W_list, b_list):
       outputs = tf.nn.xw_plus_b(outputs, W, b_list[idw])
       outputs = self.activation(outputs)
     return outputs
+
+  def none_tensors(self):
+    W_list, b_list = self.W_list, self.b_list
+    self.W_list, self.b_list = [], []
+    out_tensor, trainable_weights, variables = self.out_tensor, self.trainable_weights, self.variables
+    self.out_tensor, self.trainable_weights, self.variables = None, [], []
+    return W_list, b_list, out_tensor, trainable_weights, variables
+
+  def set_tensors(self, tensor):
+    self.W_list, self.b_list, self.out_tensor, self.trainable_weights, self.variables = tensor
+
+
+class MessagePassing(Layer):
+  """ General class for MPNN
+  default structures built according to https://arxiv.org/abs/1511.06391 """
+
+  def __init__(self,
+               T,
+               message_fn='enn',
+               update_fn='gru',
+               n_hidden=100,
+               **kwargs):
+    """
```

Comment:
The indentation here looks a little weird. Maybe try updating `yapf` to the latest version to see if the artifact vanishes.